### DISCUSSION TITLE
css outline issue...how to disable the focus outline of the '.container' inside the shadow root...

### DISCUSSION THREAD

**Discussion by rchrdnsh** [google/model-viewer#1921] [Q&A]
The `div` with the `class='container' adds an outline when `:focused`, which is not nice to look at. How can I disable that outline and maybe replace it with something nicer?

<img width="1440" alt="Screen Shot 2020-09-23 at 9 12 15 PM" src="https://user-images.githubusercontent.com/7884263/94100075-7758d200-fde1-11ea-9b3b-a2a8f123dac9.png">

**Comment by elalish** [comment]
We recommend the :focus-visible polyfill for just this purpose.

**Comment by rchrdnsh** [comment]
k, installed the polyfill, but not sure how to use it...do I need to do anything at all to `<model-viewer>` or any of it's child components? do I need to target the shadow `.container`, or should it simply just work?

**Comment by elalish** [comment]
Should just work. You can test on our [Glitch](https://glitch.com/edit/#!/model-viewer).
